guides/gatherers.md

```diff
@@ -197,6 +198,55 @@ facts:
 
 For extra information refer to [trento-project/agent/../gatherers/sbd_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbd_test.go)
 
+### sbd_dump
+
+This gatherer allows accessing the dumped data about all the configured devices on target nodes.
+It executes `sbd -d <device> dump` for all the devices in `/etc/sysconfig/sbd` and aggregates results in only one fact.
+
+Note that:
+- no arguments are required and any provided argument would just be ignored.
+- if any dump fails, a fact error is returned
+
+```rust
```

Comment:
Maybe put a header here saying that the "keys" are sanitized and this is how they can access the values in `rhai`

Reply:
Added the line above
> Dumped keys (`Timeout (watchdog)`, `Timeout (msgwait)`, `Number of slots`, etc) are sanitized to simplify their access and usage in the expression language.

How does that work?